### ISSUE TITLE
Symfony 4.2 compatibility

### ISSUE THREAD

**Issue by komik966** [doctrine/DoctrineBundle#883]
Error:
```
TypeError: Argument 1 passed to Symfony\Bridge\Doctrine\PropertyInfo\DoctrineExtractor::__construct() must implement interface Doctrine\Common\Persistence\Mapping\ClassMetadataFactory, instance of Doctrine\ORM\EntityManager given
```

Possible cause:
https://github.com/doctrine/DoctrineBundle/blob/82d2c63cd09acbde2332f55d9aa7b28aefe4983d/DependencyInjection/DoctrineExtension.php#L801-L803

**Comment by dunglas** [comment]
Can you run `composer info` please? It looks like you use DoctrineBridge 4.1 with PropertyInfo 4.2.

**Comment by komik966** [comment]
This was the problem. Thanks

**Comment by dunglas** [comment]
@komik966 can you try if https://github.com/doctrine/DoctrineBundle/pull/884 fixes your problem when using DoctrineBridge 4.1 with PropertyInfo 4.2 please?

**Comment by komik966** [comment]
@dunglas Tested, works good, error disappeared
